diff --git a/dist/index.mjs b/dist/index.mjs
index 1a5c1aba142e06238f99ab061b80e8c46520f3f8..49bf78ce79d8410b4a048aa985ad04827c0a4ac1 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -313,7 +313,7 @@ function GlobalModeBuildPlugin({ uno, ready, extract, tokens, filter, getConfig
     {
       name: "unocss:global:build:generate",
       apply(options, { command }) {
-        return command === "build" && !options.build?.ssr;
+        return command === "build" && options.build?.ssr;
       },
       enforce: "post",
       async generateBundle(options, bundle) {
